ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml

```diff
@@ -45,6 +45,7 @@ Build_Tool_Packages:
   - perl-GD
   - perl-libwww-perl
   - perl-Time-HiRes
+  - openssh-clients               # IBM: cloning over SSH
```

Comment:
Just realised - this is also not in the right place alphabetically - sorry to be picky but can we move it to just before `openssl-devel` please?

Reply:
Right, good catch. I had it there as `ssh` before I found the right package name.